packages/core/src/App/Containers/RealityCheckModal/reality-check-modal.jsx

```diff
@@ -78,6 +89,12 @@ const RealityCheckModal = ({
         setRealityCheckDuration(values.interval);
     };
 
+    const handleIntervalInputMobileFocus = ref => {
+        setTimeout(() => {
```

Comment:
why do we need setTimeout?

Reply:
To make sure keyboard is **open**.

Reply:
can we instead wait for `is_onscreen_keyboard_active` to be true

``` 
await when(() => is_onscreen_keyboard_active);
```

Reply:
Actually `is_onscreen_keyboard_active` doesn't wait for keyboard animation to be finished. It turns `true` as soon as users touch an input field. So if we scroll to the input before keyboard animation is finished, scrolling will be useless. We need a time gap here.

Reply:
Ah okay 👍